subgraph/tools/lib/graph.sh

```
@@ -0,0 +1,419 @@
+        --ipfs "$REMOTE_IPFS_NODE" \
+        "$graph_name" \
+        "$SUBGRAPH_YAML"; then
+        log_error "Failed to deploy subgraph locally"
+        return $EXIT_ERROR
+    fi
+    
+    log_success "Subgraph deployed successfully!"
+    log_info "  Access your subgraph at: ${LOCAL_GRAPH_NODE}/subgraphs/name/${graph_name}"
+    
+    return $EXIT_SUCCESS
+}
```

Shellcheck (0.10.0)

[warning] deploy_subgraph_local references arguments, but none are ever passed.

(SC2120)